test/Graphics/ConvexShape.test.cpp

```diff
@@ -42,4 +45,52 @@ TEST_CASE("[Graphics] sf::ConvexShape")
         convex.setPoint(0, {3, 4});
         CHECK(convex.getPoint(0) == sf::Vector2f(3, 4));
     }
+
+    SUBCASE(
+        "Construct clockwise ConvexShapes from CircleShapes to verify that they get approx. the same geometric center")
+    {
+        sf::ConvexShape convex;
+        for (unsigned int i = 2; i < 10; ++i)
+        {
+            sf::CircleShape circle(4.f, i);
+            convex.setPointCount(i);
+            for (unsigned j = 0; j < i; ++j)
+            {
+                convex.setPoint(j, circle.getPoint(j));
+            }
+            CHECK(convex.getGeometricCenter() == Approx(circle.getGeometricCenter()));
+        }
+    }
+
+    SUBCASE(
+        "Construct counterclockwise ConvexShapes from CircleShapes to verify that they get approx. the same geometric "
```

Comment:
Why do this same thing clockwise and counterclockwise?

Reply:
@ChrisThrasher Some of the algorithms people use need to detect if the points are stored in clockwise or counterclockwise order and do a unary `-` on the result at the end. It's just a precaution if someone wants to optimize/replace the algorithm in the future.